From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ZHANG Yuntian <yuntian@protoducer.com>
Date: Tue, 13 Sep 2022 00:19:20 +0800
Subject: [PATCH] Update build dependencies

---
 debian/control | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/control b/debian/control
index 45e9ab9..e3634e7 100644
--- a/debian/control
+++ b/debian/control
@@ -34,6 +34,7 @@ Build-Depends: autotools-dev,
                libusbredirparser-dev (>= 0.6-2),
                libvirglrenderer-dev,
                libzstd-dev,
+               lintian,
                meson,
                python3-minimal,
                python3-sphinx,
-- 
2.37.2

